### DISCUSSION TITLE
现在同步区块高度太慢了，没法追上，20秒同步一个块，全网区块高度每6秒增加1个。

### DISCUSSION THREAD

**Discussion by k178613133** [filecoin-project/lotus#4625] [Storage Provider] [answered]
如题：按照这样，那不是永远都没法追上了，请问大家同步一个块需要多久时间？有同步速度很快的吗？

前几天同步的时候很快就同步完成了，现在是参与的人太多了，大家都同步的慢，还是我个人的问题呢？

请各位大佬说一下自己同步一个快需要多少时间，谢谢了。

**Comment by CD-oursoft** [comment, marked as answer]
用ssd硬盘存储，可以解决，我以前也慢，就是因为我存储在机械硬盘，把louts移到ssd，问题就解决了

**Comment by whyrusleeping** [comment]
Please file issues in english. None of the devs speak chinese.
